bin/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 90,
+  "singleQuote": true,
+  "bracketSpacing": false,
+  "trailingComma": "none",
```

Comment:
I've added another prettier config which sets `trailingComma` to `none` so we don't break compat with older node.